projects/step-frontend/src/lib/modules/artefacts/artefacts.module.ts

```diff
@@ -72,14 +78,14 @@ export class ArtefactsModule {
     this._artefactService.register('TestCase', {
       icon: 'glyphicon-list-alt',
       iconNg2: 'list',
-      form: 'partials/artefacts/testCase.html',
+      component: TestCaseComponent,
```

Comment:
Are these required or would it also work without them?

Reply:
It can be removed. I will remove it with the next commit